opm/parser/eclipse/EclipseState/Tables/Tables.cpp

```diff
@@ -67,6 +67,7 @@
 #include <opm/parser/eclipse/EclipseState/Tables/SwofTable.hpp>
 #include <opm/parser/eclipse/EclipseState/Tables/TableContainer.hpp>
 #include <opm/parser/eclipse/EclipseState/Tables/WatvisctTable.hpp>
+#include <opm/parser/eclipse/EclipseState/Tables/DensityTable.hpp>
```

Comment:
well, the confusion is probably in my side: when I originally wrote these classes I intended them to represent continuous (1D or 2D) functions. since DENSITY represents just single data points (?), this keyword does not fall into this definition. (my original mental model may have become obsolete since then, though.) Anyway, I don't think this is a big deal.